larq_compute_engine/tflite/kernels/bconv2d_impl.h

```diff
@@ -136,31 +147,32 @@ inline void BConv2D(const ConvParams& params, const RuntimeShape& input_shape,
       static std::vector<TBitpacked> input_data_bp_buffer;
       // The input tensor has this shape which we bitpack along the channels
       // dimension [batch, input height, input width, channels].
-      ce::core::packbits_tensor(input_shape, input_data, packed_input_shape,
-                                input_data_bp_buffer);
+      ce::core::packbits_tensor(input_shape, input_data, params.input_offset,
```

Comment:
what is `input_offset`?

Reply:
Its `zero_point`, but for some structs, tflite renamed it to `offset` so we can't change that name unfortunately.